srtcore/core.cpp

```diff
@@ -7927,21 +7927,22 @@ int srt::CUDT::sendCtrlAck(CPacket& ctrlpkt, int size)
     // The TSBPD thread may change the first lost sequence record (TLPKTDROP).
     // To avoid it the m_RcvBufferLock has to be acquired.
     UniqueLock bufflock(m_RcvBufferLock);
-
+    if(m_bBufferWasFull && getAvailRcvBufferSizeNoLock() > 0)
+    {
+        sendAckAgain = true;
+    }
     int32_t ack;    // First unacknowledged packet sequence number (acknowledge up to ack).
     if (!getFirstNoncontSequence((ack), (reason)))
         return nbsent;
 
-    if (m_iRcvLastAckAck == ack)
+    if (m_iRcvLastAckAck == ack && !sendAckAgain)
     {
-        HLOGC(xtlog.Debug,
-              log << CONID() << "sendCtrl(UMSG_ACK): last ACK %" << ack << "(" << reason << ") == last ACKACK");
-        return nbsent;
+    return nbsent;
```

Comment:
```suggestion
        return nbsent;
```

Reply:
```c++
 HLOGC(xtlog.Debug,
              log << CONID() << "sendCtrl(UMSG_ACK): last ACK %" << ack << "(" << reason << ") == last ACKACK");
``` 
I removed it but now that I think about it it is better to keep it. Identation should be correct in my next push as well